.github/workflow/test.yml

```diff
@@ -0,0 +1,52 @@
+name: Run tests
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+jobs:
+    test:
+      runs-on: ubuntu-latest
+      strategy:
+        matrix:
+          python: [2.7, 3.6, 3.7]
```

Comment:
Please include 3.8, 3.9, and 3.10 in these matrices. This may also require updates to tox.ini